bin/CoreData/Shaders/GLSL/v2/P_HSV.glsl

```diff
@@ -0,0 +1,55 @@
+#include "_Config.glsl"
+#include "_Uniforms.glsl"
+#include "_VertexLayout.glsl"
+#include "_VertexTransform.glsl"
+#include "_VertexScreenPos.glsl"
+#include "_Samplers.glsl"
+
+VERTEX_OUTPUT_HIGHP(vec2 vScreenPos)
+
+#ifdef URHO3D_PIXEL_SHADER
+    UNIFORM_BUFFER_BEGIN(6, Custom)
+        UNIFORM(mediump vec4 cHSVParams)
+    UNIFORM_BUFFER_END(6, Custom)
+#endif
+
+#ifdef URHO3D_VERTEX_SHADER
+void main()
+{
+    VertexTransform vertexTransform = GetVertexTransform();
+    gl_Position = WorldToClipSpace(vertexTransform.position.xyz);
+    vScreenPos = GetScreenPosPreDiv(gl_Position);
+}
+#endif
+
+#ifdef URHO3D_PIXEL_SHADER
+
+// You can find good explanation of the math here:
+// https://blog.en.uwa4d.com/2022/09/29/screen-post-processing-effects-color-models-and-color-grading/
+
+vec3 RGBToHSV(vec3 c)
```

Comment:
Consider doing half-math so it is more mobile-friendly. Just be careful and don't replace-all: half cannot be used as constructor. So, half4 var = vec4(...).